### PR TITLE
Writefile fixes

### PR DESCRIPTION
**Pull request by molhamkhalil**

### CODE REVIEW

**Comment by jonathanudd** [comment]
The files you have changed are missing an empty line at the end of the files.

**Comment by jonathanudd** [comment]
I think the `tsconfig.json` file still should be in the server folder. Did you say you had problems with debugging otherwise?

**Comment by molhamkhalil** [comment]
yep, the debugging did not working at all untill I did this modification!

Den 2016-02-08 kl. 10:21, skrev Jonathan Udd:

File: server/source/Backend_User.ts [hunk 1]
```diff
@@ -1,5 +1,3 @@
-/// <reference path="../../typings/node/node.d.ts" />
-
 module Exclusive {
 	export class Backend_User {
 		private company: string;
```
Comment (fredrikbryntesson) [inline server/source/Backend_User.ts:3]: Backend_User? Remove underscore.

**Comment by molhamkhalil** [comment]
I removed all of them yesterday!

Den 2016-02-08 kl. 10:18, skrev Jonathan Udd:

File: server/source/Backend_User.ts (hunk 1 above)
Comment (molhamkhalil) [inline server/source/Backend_User.ts:3]: done.

**Comment by jonathanudd** [comment]
There should be an empty line at the end of each file. This is according to the POSIX standard and some programs expect this.

**Comment by molhamkhalil** [comment]
mmmmm, okay... I am adding them now

Den 2016-02-08 kl. 11:00, skrev Jonathan Udd:

**Pull request merged**